### PR TITLE
Themes.md: Show a table of themes instead of a long list

### PR DESCRIPTION
**Pull request by scorphus**
Ref #806

### CODE REVIEW

**Comment by scorphus** [comment]
@bobthecow: Sorry for the extra noise, I guess I created the PR too soon.

I experimented with the number of columns and think 7 is okay, but that's MO. I'd love to read your thoughts.

File: docs/Themes.md [hunk 1]
```diff
@@ -1,100 +1,142 @@
...
-- [will](#will)
-- [wolf-theme](#wolf-theme)
-- [yimmy](#yimmy)
-- [zeit](#zeit)
-- [zephyr](#zephyr)
-- [zish](#zish)
-
-# agnoster
+
+<!-- ATTENTION: This file is auto-generated! Changes will be lost! -->
```
Comment (bobthecow) [inline docs/Themes.md:3]: 🎉

**Review by bobthecow — APPROVED** [review]
:shipit:

**Comment by bobthecow** [comment]
> I experimented with the number of columns and think 7 is okay, but that's MO. I'd love to read your thoughts.

Yeah, that looks good to me. We can always adjust later if we need to. Thanks!

**Pull request merged**